docs/concepts/artifact.mdx

```diff
@@ -167,26 +193,22 @@ as per the  [empty descriptors guidance][empty-descriptor].
 
 ### Deciding on artifactType or config.mediaType
```

Comment:
Since `artifactType` is always recommended in all leaf nodes, should we change the title to focus more on config descriptor and media type?